test-server/test-echo.c

```diff
@@ -33,6 +33,9 @@
 #include <syslog.h>
 #include <sys/time.h>
 #include <unistd.h>
+#else
+#include "../win32port/win32helpers/gettimeofday.h"
```

Comment:
Also... Why have you even added this include? None of your changes uses ?`gettimeofday`?